CHANGELOG.md

```diff
@@ -3031,6 +3038,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Uncategorized
 - Added the ability to restore accounts from seed words.
 
+[Unreleased]: https://github.com/MetaMask/metamask-extension/compare/v10.16.1...HEAD
+[10.16.1]: https://github.com/MetaMask/metamask-extension/compare/v10.15.1...v10.16.1
 [Unreleased]: https://github.com/MetaMask/metamask-extension/compare/v10.16.0...HEAD
```

Comment:
This should fix the changelog validation:
```suggestion
[10.16.1]: https://github.com/MetaMask/metamask-extension/compare/v10.16.0...v10.16.1
```

Reply:
thanks